packages/docs/stories/components/select.stories.ts

```diff
@@ -0,0 +1,404 @@
+    </syn-select>
+  `,
+};
+
+export const PrefixSuffixTextAndIcons: Story = {
+  name: 'Prefix Suffix Text and Icons',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'prefix'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select placeholder="Small" size="small" clearable>
+      <syn-icon name="wallpaper" slot="prefix"></syn-icon>
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+    <br />
+    <syn-select placeholder="Medium" size="medium" clearable>
+      <syn-icon name="wallpaper" slot="prefix"></syn-icon>
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+    <br />
+    <syn-select placeholder="Large" size="large" clearable>
+      <syn-icon name="wallpaper" slot="prefix"></syn-icon>
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+  `,
+};
+
+export const CustomTags: Story = {
+  name: 'Custom Tags',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'gettag'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select
+      placeholder="Select one"
+      value="option email phone"
+      multiple
+      clearable
+      class="custom-tag"
+    >
+      <syn-option value="option">Option</syn-option>
+      <syn-option value="email">
+        <syn-icon slot="prefix" name="mail_lock"></syn-icon>
+        Email
+      </syn-option>
+      <syn-option value="phone">
+        <syn-icon slot="prefix" name="phone_callback"></syn-icon>
+        Phone
+      </syn-option>
+      <syn-option value="chat">
+        <syn-icon slot="prefix" name="chat_bubble"></syn-icon>
+        Chat
+      </syn-option>
+    </syn-select>
+
+    <script type="module">
+      const select = document.querySelector('.custom-tag');
+
+      select.getTag = (option, index) => {
+        // Use the same icon used in the <syn-option>
+        const optionElement = option.querySelector('syn-icon[slot="prefix"]');
+        
+        if (!optionElement) {
+          return \`
+          <syn-tag removable>
+          \${option.getTextLabel()}
+          </syn-tag>
+          \`;
+        }
+        
+        const { name } = optionElement;
+
+        // You can return a string, a Lit Template, or an HTMLElement here
+        return \`
+          <syn-tag removable>
+            <syn-icon name="\${name}"></syn-icon>
+            \${option.getTextLabel()}
+          </syn-tag>
+        \`;
+      };
+    </script>
+  `,
+};
+
+// Bundled screenshot story
+export const Screenshot: Story = generateScreenshotStory([
+  Labels,
```

Comment:
Yesterday we decided, that the default story should also be added for each component to the screenshot story. Otherwise it can happen, that no default state can be checked by design.
```suggestion
  Default,
  Labels,
```